### DISCUSSION TITLE
Main objective

### DISCUSSION THREAD

**Discussion by jog1t** [jog1t/aye-bucko#31] [Gameplay ideas]
Get on a ship or pick a treasure chest?

**Comment by aweczet** [comment]
How about a racing parkour game? 2-4 players will compete to win. 
Objective is to pick up a treasure chest and deliver it to the ship.
As a player you will start on the ship. Treasure chest will be spawned in one of few locations (randomly). If you fall from path you will be respawned at starting locations. If you fall while holding a chest, it will be respawned in different location.
It would be nice if you had some way to push other players off path.

**Reply by jog1t** [reply]
We can go with multiple types of gameplay. Like https://store.steampowered.com/search/collections/steam-codes?developer=Jackbox%20Games%2C%20Inc.. Let’s start with a parkour one. Later we can add a lobby where players will have an option to choose which genres they want to play. What do you think?

**Reply by aweczet** [reply]
Sounds good. Another example: https://store.steampowered.com/app/1097150/Fall_Guys_Ultimate_Knockout/
I agree with starting with a parkour one. It seems quite easy to implement and to test.

**Reply by jog1t** [reply]
Ticket: https://github.com/jog1t/aye-bucko/issues/34

**Comment by jog1t** [comment]
Another idea that comes to my mind is to have a simple PvP mode, in which players will have couple of maps to play on with simple objective - kill other players. 

Maybe to spice it up, we could have one weapon on a map - the deadly dagger. Maybe other players will have an option to place traps or something else? 🤔

**Reply by aweczet** [reply]
Yeah, it might looked a little bit like Brawlhalla(1) or Stick Fight: The Game(2) with this one deadly weapon on the map. However I think that it would be best to stick with simplest idea first (getting online fighting just right, might be a little tricky imo).

(1) Brawlhalla: https://store.steampowered.com/app/291550/Brawlhalla/
(2) Stick Fight: The Game: https://store.steampowered.com/app/674940/Stick_Fight_The_Game/
